akka-docs/src/main/paradox/stream/operators/Partition.md

```diff
@@ -6,11 +6,21 @@ Fan-out the stream to several streams.
 
 ## Signature
 
+@apidoc[Partition]
```

Comment:
I didn't find any other usage of `@signature` for graph operators, maybe this isn't too bad (and low effort)